bin/katello-certs-check

```diff
@@ -152,11 +152,13 @@ function check-priv-key () {
 function check-ca-bundle () {
     printf "Checking CA bundle against the certificate file: "
     CHECK=$(openssl verify -CAfile $CA_BUNDLE_FILE -purpose sslserver -verbose $CERT_FILE 2>&1)
-    if [[ $? == "0" ]]; then
-        success
+
+    ERROR_PATTERN="error [0-9]+ at"
+    if [[ $CHECK =~ $ERROR_PATTERN || $? != "0" ]]; then
```

Comment:
I'm always cautious with using `$?` further from the actual execution since it's easy to break in refactoring. Could you define `ERROR_PATTERN` before `CHECK=...` or do it inline?

Reply:
I'm not sure if my change fully achieves what you are wanting.

Reply:
I normally always keep the `$()` line and `$?` without any whitespace separation. My reasoning is that `$?` is a side effect of `$()` and side effects should always be minimized. If they're not together, someone might add in another command in between. I'd also move it to the front to keep it as close as possible.
```suggestion
    if [[ $? != "0" || $CHECK =~ $ERROR_PATTERN ]]; then
```